### DISCUSSION TITLE
how to set maxClauseCount?

### DISCUSSION THREAD

**Discussion by climberwb** [castorini/pyserini#969] [Q&A] [answered]
I am getting an issue with pyserini:  JVM exception occurred: maxClauseCount is set to 1024. How can I increase this value?

**Comment by lintool** [comment, marked as answer]
`BooleanQuery.setMaxClauseCount` is the method that changes the value, but this isn't exposed in Pyserini. What types of queries are you submitting that bumps you up against this limit?
